src/components/organisms/proposal-summary.tsx

```diff
@@ -1,4 +1,13 @@
-import type { UIGroup, UIProposal, Vote, VoteOptionType } from 'types'
+import { Any } from '@haveanicedavid/regen-ts/types/codegen/google/protobuf/any'
```

Comment:
You should be able to import `Any` from `types` as well (it's re-exported in `golang.types.ts`

Reply:
https://github.com/regen-network/groups-ui/pull/66/commits/f740345783fdcf2dbfac11c58f1c4ebd88030837